src/structures/gdt.rs

```diff
@@ -105,6 +105,38 @@ impl GlobalDescriptorTable {
         }
     }
 
+    /// Forms a GDT from a slice of `u64`.
+    ///
+    /// # Safety
+    ///
+    /// * The user must make sure that the entries are well formed
+    /// * The provided slice **must not be larger than 8 items** (only up to the first 8 will be observed.)
+    #[inline]
+    #[cfg(feature = "nightly")]
+    pub const unsafe fn from_raw_parts(slice: &[u64]) -> GlobalDescriptorTable {
```

Comment:
Given that we no longer have multiple "parts", I think the following name would fit better now:

```suggestion
    pub const unsafe fn from_raw_slice(slice: &[u64]) -> GlobalDescriptorTable {
```